app/components/file-browser/component.ts

```diff
@@ -95,7 +95,7 @@ export default class FileBrowser extends Component.extend({
     isMoving = false;
     loaded = true;
     uploading = A([]);
-    clickable = ['.dz-upload-button'];
+    clickable = ['.dz-upload-button', '.file-browser-empty'];
```

Comment:
Probably going to get merge conflicts with #162

Reply:
Oh wow yeah I messed up my branches this should be here. Good catch @binoculars